Sources/FigSchema/FigScriptGenerator.swift

```diff
@@ -1,24 +1,31 @@
 import Foundation
 
-public struct FigScriptGenerator {
-    static let encoder: JSONEncoder = {
-        let encoder = JSONEncoder()
+public class FigScriptGenerator {
+    static let shared = FigScriptGenerator()
```

Comment:
I didn't want to make the generator a singleton since we might want to add customisation options to `init` in the future. Perhaps we can rename this to `default` to imply that this instance is configured with the default options (ie none at present). Regardless, this should probably be made public since FigScriptGenerator itself is a public API and otherwise there's no way to obtain an instance.
```suggestion
    public static let `default` = FigScriptGenerator()
```